projects/plugins/protect/src/class-threats.php

```diff
@@ -81,4 +81,43 @@ public static function ignore_threat( $threat_id ) {
 		return self::update_threat( $threat_id, array( 'ignore' => true ) );
 	}
 
+	/**
+	 * Scan enqueue
+	 *
+	 * @return bool
+	 */
+	public static function scan() {
+		$blog_id      = Jetpack_Options::get_option( 'id' );
+		$is_connected = ( new Connection_Manager() )->is_connected();
+
+		if ( ! $blog_id || ! $is_connected ) {
+			return new WP_Error( 'site_not_connected' );
```

Comment:
```suggestion
			return false;
```
This method's `@return` documentation suggests it should only return true/false.